MapboxSearch/autofill/build.gradle

```diff
@@ -84,7 +84,7 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-engine:$junit_5_version"
     testImplementation project(":common-tests")
 
-    androidTestImplementation "com.mapbox.common:okhttp:$common_sdk_version"
+    androidTestImplementation "com.squareup.okhttp3:okhttp:4.12.0"
```

Comment:
@DzmitryFomchyn @ahmedaly16 
I guess we need to republish this package, am I wrong here?

Reply:
No, we can't republish this because we don't control it, it's another team responsibility.

@eugenes78 try 

`okhttp_mock_version = '1.5.0'`
...
`testImplementation "com.github.gmazzo:okhttp-mock:$okhttp_mock_version"`

Reply:
Resolved through a new variable called `okhttp_version`